js/private/pnpm_utils.bzl

```diff
@@ -91,14 +91,14 @@ pnpm_utils = struct(
     friendly_name = _friendly_name,
     virtual_store_name = _virtual_store_name,
     strip_peer_dep_version = _strip_peer_dep_version,
-    # Prefix namespace to use for generated js_binary targets and aliases
-    js_package_target_namespace = "jsp__",
```

Comment:
looks ugly breaking change for my downstream projects, but now's the time

Reply:
indeed